logstreams/pom.xml

```diff
@@ -87,6 +84,11 @@
       <groupId>io.prometheus</groupId>
       <artifactId>simpleclient</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.zeebe</groupId>
+      <artifactId>atomix-cluster</artifactId>
+      <version>0.23.0-SNAPSHOT</version>
```

Comment:
We don't need the version here I think, since we should add `atomix-cluster` in our Zeebe BOM